widgets/pulseaudio.lua

```diff
@@ -31,7 +31,15 @@ local function factory(args)
     --  '/volume:/ p' prints the line containing 'volume:'.
     --  '/muted:/ p' prints the line containing 'muted:'.
     --  '/device\\.string/ p' prints the line containing 'device.string:'.
-
+    
+    local settings    = args.settings or function()
```

Comment:
The `settings` function is intended to be always defined, if you use the `.widget`.